### PR TITLE
Update vendors & fix travis issues

### PR DESCRIPTION
**Pull request by douaejeouit**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -31,7 +31,7 @@ commit:
 
 .PHONY: update-deps
 update-deps:
-	glide update --no-recursive --strip-vcs
```
Comment (marminthibaut) [inline Makefile]: Are you sure you want to do this? Why it does not work with `--no-recursive --strip-vcs`

Did you try to keep at least the `--strip-vcs` option?

**Comment by marminthibaut** [comment]
`make` does not work on my environment. Is it ok in yours?

make output:

```
$ make 
glide update  --no-recursive
[INFO] Downloading dependencies. Please wait...
[INFO] Fetching updates for github.com/elastic/beats.
[WARN] github.com/elastic/beats is an empty directory. Fetching a new copy of the dependency.
[INFO] Fetching updates for github.com/stretchr/testify.
[WARN] github.com/stretchr/testify is an empty directory. Fetching a new copy of the dependency.
[INFO] Fetching updates for github.com/fsouza/go-dockerclient.
[WARN] github.com/fsouza/go-dockerclient is an empty directory. Fetching a new copy of the dependency.
[INFO] Setting version for github.com/fsouza/go-dockerclient to e40d8323830a4ba217ba25af3b12cf9646717be9.
[INFO] Setting version for github.com/elastic/beats to 05cd6641cf6e4fa12caa6e85588e181f8eefe1e6.
[WARN] Skipping lockfile generation because full dependency tree is not being calculated
make update
make[1]: Entering directory '/home/tmarmin/.go/src/github.com/ingensi/dockerbeat'
bash ./vendor/github.com/elastic/beats/libbeat/scripts/update.sh dockerbeat github.com/ingensi/dockerbeat ./vendor/github.com/elastic/beats/libbeat
Beat name: dockerbeat
Beat path: ../dockerbeat
Start modifying beat
Update config file
Update fields
Traceback (most recent call last):
  File "./vendor/github.com/elastic/beats/libbeat/scripts/generate_fields_docs.py", line 1, in <module>
    import yaml
ImportError: No module named yaml
vendor/github.com/elastic/beats/libbeat/scripts/Makefile:207: recipe for target 'update' failed
make[1]: *** [update] Error 1
make[1]: Leaving directory '/home/tmarmin/.go/src/github.com/ingensi/dockerbeat'
Makefile:14: recipe for target 'init' failed
make: *** [init] Error 2
```

**Comment by douaejeouit** [comment]
@marminthibaut ,, i think it's normal to get this error  if you cloned the prject on updates branch, it said that the directory is empty and i just noticed that the vendors are empty on the branch! I don't why it's the case. Please try to run glide update before you run the make cmd, it should fetch the right packages. I'll be on it tomorrow and i'll keep u updated.

**Comment by marminthibaut** [comment]
IMO update should be automatic...

**Comment by douaejeouit** [comment]
Every thing is  working as expected for me. I'll se it as soon as possible.

**Comment by marminthibaut** [comment]
OK so can you post here the build procedure from a clean cloned repository?

Thanks

**Comment by douaejeouit** [comment]
@marminthibaut , when you clone the project to your local machine and you run  the make, it runs glide update --no-recursive, with this option, you disable updating dependencies. The problem comes from the updated version of docker client, it has some dependencies wihch have to be fetched so that the project compiles correctly. so by runing glide with that option, not all the dependencies are fetched and naturally its KO since the compilateur can't find packages the project needs !!  
I'm not sure about the new version of docker client .. Are you sure you want to update  it ? !
Please try with my last commit, it should be Ok for you!

**Comment by marminthibaut** [comment]
it closes #118

**Pull request merged**